pkg/event_handler/action_triggered_handler.go

```diff
@@ -49,7 +49,7 @@ func (eh ActionTriggeredHandler) HandleEvent() error {
 		eh.Logger.Error(msg)
 		err = eh.sendEvent(keptnv2.GetFinishedEventType(keptnv2.ActionTaskName),
 			eh.getActionFinishedEvent(keptnv2.ResultFailed, keptnv2.StatusErrored, *actionTriggeredEvent, msg))
-		return errors.New(msg)
+		return nil
```

Comment:
I think just returning `nil` is a very one-sided interpretation of #76  :D 
I'd suggest you check if `sendEvent` returns an error, and then process that error + wrap the original error